installer/kubernetes/tasks/main.yml

```diff
@@ -0,0 +1,125 @@
+---
+- name: Set the Kubernetes Context
+  shell: "kubectl config set-context {{ kubernetes_context }}"
+
+- name: Get Namespace Detail
+  shell: "kubectl get namespace {{ awx_kubernetes_namespace }}"
+  register: namespace_details
+  ignore_errors: yes
+
+- name: Get Postgres Service Detail
+  shell: "kubectl describe svc awx-postgresql -n {{ awx_kubernetes_namespace }}"
+  register: postgres_svc_details
+  ignore_errors: yes
+  when: "pg_hostname is not defined or pg_hostname == ''"
+
+- name: Create AWX Kubernetes Project
+  shell: "kubectl create namespace {{ awx_kubernetes_namespace }}"
+  when: namespace_details.rc != 0
+
+# TODO: This is duplicated in the openshift role, probably needs to be moved to the image_build role
+- name: Manage AWX Container Images
+  block:
+    - name: Get docker registry password from oc if needed
+      block:
+        - name: Set docker registry password
+          shell: oc whoami -t
```

Comment:
Surprised to see an `oc` command in this playbook. Copy / paste error?

Reply:
yyyes it is.